code/modules/food_and_drinks/food/seafood.dm

```diff
@@ -25,25 +25,25 @@
 	trash = /obj/item/trash/plate
 	filling_color = "#7A3D11"
 	bitesize = 3
-	list_reagents = list("nutriment" = 4, "sodiumchloride" = 1, "blackpepper" = 1)
+	list_reagents = list("nutriment" = 4, "vitamin" = 2)
 
 /obj/item/weapon/reagent_containers/food/snacks/catfishmeat
 	name = "raw catfish"
 	desc = "A fillet of raw catfish"
 	icon = 'icons/obj/food/seafood.dmi'
 	icon_state = "fishfillet"
 	filling_color = "#FFDEFE"
-	bitesize = 3
-	list_reagents = list("protein" = 3)
+	bitesize = 6
+	list_reagents = list("protein" = 3, "vitamin" = 2)
 
 /obj/item/weapon/reagent_containers/food/snacks/fishfingers
 	name = "Fish Fingers"
 	desc = "A finger of fish."
 	icon = 'icons/obj/food/seafood.dmi'
 	icon_state = "fishfingers"
 	filling_color = "#FFDEFE"
-	bitesize = 3
-	list_reagents = list("nutriment" = 4, "carpotoxin" = 3)
+	bitesize = 1
+	list_reagents = list("nutriment" = 4)
```

Comment:
There's not good criterion at the moment to be honest; most of these values are what TG utilizes. For the most part, from what I could see it was fairly logical, but I don't doubt there's some weird outliers.